### ISSUE TITLE
Add log out functionality

### ISSUE THREAD

**Issue by LorenDB** [xtreamsrl/clockify-nlp-bot#106]
I'd like to have the ability to log out of the bot. This should be doable by saying "log out", whereupon the bot could either directly log you out or ask "Are you sure?"

**Comment by LorenDB** [comment]
Any progress being made on this?

**Comment by swissbyte** [comment]
Yes, i have implemented this feature within the pull request #122 
Feel free to checkout the main branch from my fork. This also contains a lot of new features ;)
